setup.cfg

```diff
@@ -11,4 +11,4 @@ exclude =
 
 [tool:pytest]
 addopts = --flake8
-flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W504
+flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W504 E401 E303 E302 E305
```

Comment:
Please! Remove the additional ignored tests.

Reply:
The removed lint tests were too nit-picky, restrictive & provided no added value.